server/services/homepageService.ts

```diff
@@ -0,0 +1,27 @@
+import { PrisonApiClient } from '../data/interfaces/prisonApiClient'
+import { RestClientBuilder } from '../data'
+import { formatDateTimeISO } from '../utils/dateHelpers'
+
+export default class HomepageService {
+  constructor(private readonly prisonApiClientBuilder: RestClientBuilder<PrisonApiClient>) {}
+
+  public async getTodaySection(clientToken: string, activeCaseLoadId: string) {
+    const [assignedRollCount, unassignedRollCount, movements] = await Promise.all([
+      await this.prisonApiClientBuilder(clientToken).getRollCount(activeCaseLoadId),
+      await this.prisonApiClientBuilder(clientToken).getRollCount(activeCaseLoadId, true),
+      await this.prisonApiClientBuilder(clientToken).getMovements(activeCaseLoadId),
+    ])
```

Comment:
```suggestion
    const prisonApiClient =  this.prisonApiClientBuilder(clientToken);
    const [assignedRollCount, unassignedRollCount, movements] = await Promise.all([
      prisonApiClient.getRollCount(activeCaseLoadId),
      prisonApiClient.getRollCount(activeCaseLoadId, true),
      prisonApiClient.getMovements(activeCaseLoadId),
    ])
```

Reply:
Don't need to await each on an await Promise.all

Reply:
Good point. Cut and paste error as I originally didn't have the `Promise.all`
Will fix.